### PR TITLE
adding the latest config to cloudinary and able to display the image

### PR DESCRIPTION
**Pull request by nfarinello**

### CODE REVIEW

**Review by Mirkica — DISMISSED** [review]

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -13,7 +13,7 @@
   </div>
 
   <div class="form-actions">
-    <%= f.button :submit, "Log in" %>
+    <%= f.button :submit, "Log in" %>s
```
Comment (gamcdonald123) [inline app/views/devise/sessions/new.html.erb:16]: @nfarinello why is there an s at the end of this line?

File: db/migrate/20240305143503_add_photos.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class AddPhotos < ActiveRecord::Migration[7.1]
+
+  def change
+add_column :users, :photos, :string
```
Comment (gamcdonald123) [inline db/migrate/20240305143503_add_photos.rb:4]: Indentation issue

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -1,2 +1,6 @@
 class UsersController < ApplicationController
+
```
[the remark this answers is not in the document]
Comment (nfarinello) [reply app/controllers/users_controller.rb:2]: done

File: app/views/devise/sessions/new.html.erb (hunk 1 above)
Comment (nfarinello) [reply app/views/devise/sessions/new.html.erb:16]: done

File: db/migrate/20240305143503_add_photos.rb (hunk 2 above)
Comment (nfarinello) [reply db/migrate/20240305143503_add_photos.rb:4]: done

**Review by nfarinello — COMMENTED** [review]
changes requested done

**Review by gamcdonald123 — APPROVED** [review]
Fab

**Pull request merged**